sandbox/tests/validators/request-validator.spec.js

```diff
@@ -0,0 +1,32 @@
+const validator = require("../../validators/request-validator")
+const validPrescriptionBundle = require("../resources/valid-bundle.json")
+const validPrescriptionBundleWithSignature = require("../resources/valid-bundle-with-signature.json")
+const {Success} = require('monet');
+
+test('it should return an array of validation error objects when given an empty json object', () => {
```

Comment:
I like the naming convention you've gone with for new tests, can this test name be updated to match?